lldb/include/lldb/Core/Debugger.h

```diff
@@ -83,7 +83,7 @@ class Debugger : public std::enable_shared_from_this<Debugger>,
     eBroadcastBitProgress = (1 << 0),
     eBroadcastBitWarning = (1 << 1),
     eBroadcastBitError = (1 << 2),
-    eBroadcastSymbolChange = (1 << 3),
+    eBroadcastBitSymbolChange = (1 << 3),
```

Comment:
Per Jim's comments this should be a private event bit that we don't want exposed in the public API. Best to rename to contain the work "Private" and possibly use bit 31:
```
eBroadcastBitPrivateSymbolChange = (1 << 31),
```

Reply:
Would we want to both remove the bit from the SB API and use the higher-order bits for private events bits like this?

Reply:
Yes, the SB API is public, so it shouldn't be there if it's private.